src/foam/nanos/auth/User.js

```diff
@@ -849,7 +849,7 @@ foam.RELATIONSHIP({
   },
   targetProperty: {
     hidden: false,
-    section: 'systemInformation',
+    section: 'administrative',
```

Comment:
@lchanmann .. there is no 'administive' section .. why did you change this? now the group cannot be seen